api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -667,7 +667,7 @@ public void getConceptSetsInWorkspaceOne() {
             .getItems();
 
     assertThat(response.size()).isEqualTo(1);
-    assertThat(response).containsAllIn(ImmutableList.of(defaultConceptSet));
```

Comment:
You did this throughout, and I like it. I think that removing the "ImmutableList" looks cleaner.